index.html

```diff
@@ -26,11 +33,14 @@
       </nav>
     </header>
 
-		<div id=veryBottom>
+<!--		<div id="veryBottom">
       <p>
-        <!-- copyright info, miscellaneous links go here -->
       </p>
     </div>
-
+-->
+    <footer>
+            <!-- copyright info, miscellaneous links go here -->
+      <h4 id="copy">&copy funPortland <span id="dateSpan"></span></h4>
```

Comment:
Check it out: the red highlight is GitHub telling you, this is a syntax error. It needs a semicolon like this:

```html
&copy;
```